library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -237,32 +228,6 @@ private int translateNetworkTypeToApiFlag(int networkType) {
         }
     }
 
-    /**
-     * Check if the download's size prohibits it from running over the current network.
-     *
-     * @return one of the NETWORK_* constants
-     */
-    private NetworkState checkSizeAllowedForNetwork(int networkType) {
```

Comment:
why was this unused though? It's a legit use case https://github.com/novoda/download-manager/blob/master/library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java#L724